pkg/engine/events.go

```diff
@@ -55,13 +55,13 @@ func cancelEvent() Event {
 
 // DiagEventPayload is the payload for an event with type `diag`
 type DiagEventPayload struct {
-	URN      resource.URN
-	Prefix   string
-	Message  string
-	Color    colors.Colorization
-	Severity diag.Severity
-	StreamID int32
-	IsStatus bool
+	URN           resource.URN
+	Prefix        string
+	Message       string
+	Color         colors.Colorization
+	Severity      diag.Severity
+	StreamID      int32
+	StatusMessage bool
```

Comment:
i do not like this name.  esp. since 'Message' is a string :)   i think 'Status' is fine as a name.

Reply:
Changed to `Ephemeral`.